ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1169,11 +1187,13 @@ void GCS_MAVLINK_Copter::handleMessage(const mavlink_message_t &msg)
 
         // send request
         if (!pos_ignore && !vel_ignore) {
-            copter.mode_guided.set_destination_posvel(pos_vector, vel_vector, !yaw_ignore, yaw_cd, !yaw_rate_ignore, yaw_rate_cds, yaw_relative);
+            copter.mode_guided.set_destination_posvelaccel(pos_vector, vel_vector, accel_vector, !yaw_ignore, yaw_cd, !yaw_rate_ignore, yaw_rate_cds, yaw_relative);
+        } else if (pos_ignore && vel_ignore && !acc_ignore) {
```

Comment:
I think we are missing this code in the MAVLINK_MSG_ID_SET_POSITION_TARGET_GLOBAL_INT handler a little lower down in this file.

Reply:
actually these lines can never be called because of the line, "if (pos_ignore && vel_ignore) {" above